commands/databases.go

```diff
@@ -2496,9 +2497,10 @@ func RunDatabaseConfigurationGet(c *CmdConfig) error {
 		"mysql": nil,
 		"pg":    nil,
 		"redis": nil,
+		"mongo": nil,
```

Comment:
```suggestion
		"mongodb": nil,
```
Lets use `mongodb` as that is the slug used to create a MongoDB cluster. 

```
$ doctl databases options engines 
Engine
mongodb
redis
mysql
pg
kafka
opensearch
```

Reply:
:+1: Updated in [5aa07c1](https://github.com/digitalocean/doctl/pull/1582/commits/5aa07c152a820210ddf7a88941e3e81185acb8a7)